packages/@webex/plugin-meetings/test/unit/spec/stats-analyzer/index.js

```diff
@@ -2113,6 +2114,37 @@ describe('plugin-meetings', () => {
           }
         });
       });
-    });
+
+      describe('CPU Information Reporting', async () => {
+        let getNumLogicalCoresStub;
+
+        beforeEach(async () => {
+          if (getNumLogicalCoresStub) {
+            getNumLogicalCoresStub.restore();
```

Comment:
I think we can remove this if block because we are already calling a restore in afterEach.

Reply:
If it's causing test failures, we can leave it as it is.

Reply:
@Shreyas281299 thanks, good idea, works without if and I believe it'd be more clearer, good catch